bretm9/genrefy
# Nd app design

### What’s this PR do?  
- Adds basic styling to the app.
- Splits the randomly generated genres to the left side of the page and the generated playlist to the right of the page.
- Adds a normalize file to better control css styling on the page.
- Adjust some classNames for easier targeting for styling.
- Sets global pixel width to 10 px to allow for easier rem measurements.
 
### Where should the reviewer start?  
- Git pull this branch.
- Work inside App and Header component folders, component and styling files.
 
### How should this be manually tested?  
- run `npm test` to open server on localhost:3000
- check initial styling to ensure it populates on page with no broken graphics.
 
### Any background context you want to provide?  
- Individual components will be styled in their own branches rather than all at once for atomic commits related to changing their files.
 
### What are the relevant tickets?  
- closes #75 
- closes #77
- closes #78
- closes #38
 
### Screenshots (if appropriate)  
- N/A
 
### Questions: 
- N/A
